app/actors/curation_concerns/actors/add_to_collection_actor.rb

```diff
@@ -22,8 +22,11 @@ def add_to_collections(new_collection_ids)
           # remove from old collections
           (curation_concern.in_collection_ids - new_collection_ids).each do |old_id|
             collection = ::Collection.find(old_id)
-            collection.members.delete(curation_concern)
-            collection.save
+            # don't remove from collections current user doesn't have edit permissions for
+            if collection.edit_users.include? user.email
```

Comment:
Was this PR merged before the email/user_key change was made? @hackmastera would you mind putting in a quick follow-up PR?